components/python/rapidfuzz/rapidfuzz-PYVER.p5m

```diff
@@ -36,7 +36,7 @@ file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/__pyinstaller/hook-ra
 file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/__pyinstaller/test_rapidfuzz_packaging.py
 file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/_common_py.py
 file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/_feature_detector.py
-file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/_feature_detector_cpp.so
+file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/_feature_detector_cpp.so variant.arch=i386
```

Comment:
Are you sure rapidfuzz works properly without this file?

Reply:
At least I was able to build python/cleo which depends on rapidfuzz.